legal-api/src/legal_api/models/party_role.py

```diff
@@ -128,12 +128,15 @@ def get_active_directors(business_id: int, end_date: datetime) -> list:
     @staticmethod
     def get_party_roles(business_id: int, end_date: datetime = None, role: str = None) -> list:
         """Return the parties that match the filter conditions."""
+        from legal_api.services import flags  # pylint: disable=import-outside-toplevel
+
         party_roles = db.session.query(PartyRole). \
             filter(PartyRole.business_id == business_id)
 
-        # TODO: rework the Officers section; change-on-change to still bring back officers
-        if not role or role != PartyRole.RoleTypes.OFFICER.value:
-            party_roles = party_roles.filter(PartyRole.role != PartyRole.RoleTypes.OFFICER.value)
+        # TODO: rework the unsupported party roles section; change-on-change to still bring back these roles
+        unsupported_roles = flags.value('unsupported-party-roles') or []
+        if not role or role.lower() not in unsupported_roles:
+            party_roles = party_roles.filter(PartyRole.role.notin_(unsupported_roles))
```

Comment:
We need to do replicate this in https://github.com/bcgov/lear/tree/main/python/common/business-registry-model

1. `entity-filer` (`business-filer`) is calling `get_party_roles`. we need to make sure flags are available in all the services  
2. I don't see any references of `flags` in `business-registry-model`. Also noticed we have a missing flag check in model. This need to be confirmed with @thorwolpert
    - Refs:
        - https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/business.py#L489
        - https://github.com/bcgov/lear/blob/main/python/common/business-registry-model/src/business_model/models/business.py#L510